### ISSUE TITLE
[3.7.0-beta] Consider adding a confirmation to save actions

### ISSUE THREAD

**Issue by wsydney76** [craftcms/cms#8094]
### Description

Now that 'Save' is the primary action, it is very easy to publish new/provisional drafts unintentionally.

I believe many people (well, at least me) are pressing ctrl+s without thinking much about it, as a habit, just with the intention to persist the current state, without wanting to initiate a workflow step. 

So a confirmation 'Do your really want to....' would be welcome.

**Comment by brandonkelly** [comment]
We debated this internally and decided against a confirmation dialog for now. However we agreed that the “Save” button is a little misleading, and doesn’t communicate that the state of the entry is going to change from a draft to a normal (published) entry.

So going forward, the button will be labeled “Create entry” instead of “Save” for unpublished drafts.

<img width="178" alt="A “Create entry” button" src="https://user-images.githubusercontent.com/47792/122830257-f3af1180-d29c-11eb-86f6-2c6dfa9f22dc.png">

The context menu action labels have been updated to use “Create” as well, and we’ve added a new “Save draft” action which saves the draft as-is and returns you to the Entries index page:

<img width="293" alt="The context menu for a “Create entry” button, which now includes “Create and continue editing” and “Create and add another” actions" src="https://user-images.githubusercontent.com/47792/122830289-fc074c80-d29c-11eb-92b7-e2abe937d206.png">

We’ve also added the draft icon to the “Status” metadata value.

<img width="214" alt="A draft’s “Status” metadata value" src="https://user-images.githubusercontent.com/47792/122830587-67511e80-d29d-11eb-8458-5a3988a678f3.png">


We think the combination of thees changes should help add clarification that the entry starts off in a draft state, and that clicking the primary submit button is going to convert it to a normal entry, as opposed to saving changes in the current draft state.
